### ISSUE TITLE
Stream pauses / breaks if you stream a Youtube video in fullscreen

### ISSUE THREAD

**Issue by makedir** [moonlight-stream/moonlight-qt#684]
Just noticed this, I am sure this worked in the past.

1. start stream to Windows 11 client via mstc.exe
2. open a youtube video / stream in Chrome
3. maximize the video
4. let mouse idle / not move
5. after a few seconds the stream freezes, though the video still plays on the server (seen on monitor)
6. moving the mouse cursor and the video stream updates randomly
7. minimizing the video and it plays normally again

Is this a bug in GFE?

**Comment by cgutman** [comment]
Yes, it seems to be an incompatibility between GFE's frame capture and some optimized video rendering techniques that some browsers like Chrome use.

Using Sunshine as a host is not affected.
